src/player.js

```diff
@@ -461,7 +463,7 @@ export default class Player extends FakeEventTarget {
         this._posterManager.setSrc(this._config.sources.poster);
         this._handleDimensions();
         this._handlePreload();
-        this._handleAutoPlay();
+        this._initAutoPlay();
```

Comment:
if we don't change anything let's keep current name

Reply:
@RoyBregman ? is this change necessary?

Reply:
No, it is not, reverted back to original name